openassessment/xblock/test/test_message.py

```diff
@@ -38,6 +38,10 @@ class TestMessageRender(XBlockHandlerTestCase):
             'completed': TODAY,
             'graded': TODAY,
         },
+        'staff': {
+            'completed': TODAY,
+            'graded': TODAY,
+        },
```

Comment:
I ought to add a test case to this file to verify that waiting for staff yields the correct message as well.